dist/error-codes.md

```diff
@@ -70,6 +71,7 @@
 | 130 | install-update: cache_type | Command failed: `php ./bin/magento cache:enable` | Command `php ./bin/magento cache:enable` runs only when Magento was installed but `./app/etc/env.php` file was absent or empty at the beginning of the deployment. Check the `cloud.log` for more information. Add `VERBOSE_COMMANDS: '-vvv'` into `.magento.env.yaml` for more detailed command output. |
 | 131 | install-update | The `crypt/key`  key value does not exist in the `./app/etc/env.php` file or the `CRYPT_KEY` cloud environment variable | This error occurs if the `./app/etc/env.php` file is not present when Magento deployment begins, or if the `crypt/key` value is undefined. If you migrated the database from another environment, retrieve the crypt key value from that environment. Then, add the value to the [CRYPT_KEY](https://devdocs.magento.com/cloud/env/variables-deploy.html#crypt_key) cloud environment variable in your current environment. See [Add the Magento encryption key](https://devdocs.magento.com/cloud/setup/first-time-setup-import-import.html#encryption-key). If you accidentally removed the `./app/etc/env.php` file, use the following command to restore it from the backup files created from a previous deployment: `./vendor/bin/ece-tools backup:restore` CLI command ." |
 | 132 |  | Can not connect to the Elasticsearch service | Check that credentials for elasticsearch are correct and service is running |
+| 133 | validate-config | Configuration for Braintree is redundant as Braintree module is not a part of Magento >= 2.4.0 | Remove CONFIG__STORES__DEFAULT__PAYMENT__BRAINTREE__CHANNEL variable from environment variables |
```

Comment:
| 133 | validate-config | Support for the Braintree module is no longer included with Magento 2.4.0 and later.  | Remove the CONFIG__STORES__DEFAULT__PAYMENT__BRAINTREE__CHANNEL variable from the variables section of the .magento.app.yaml file. For Braintree  support, use the official Braintree Payments extension from the Magento Marketplace instead. |

Reply:
This file will be updated automatically from schema.error.yaml

Reply:
@oshmyheliuk  I think the dist/error-codes.md file doesn't need to be updated because these files will be auto-generated with the most current descriptions on release.  Is that correct?